backend/fms_core/services/experiment_run.py

```diff
@@ -89,12 +92,31 @@ def create_experiment_run(experiment_run_name,
     return (experiment_run, errors, warnings)
 
 def launch_experiment_run(pk):
```

Comment:
We should rename to something closer to what it actually does. generate_run_info_file(experimentrun_id). I would recommend this service also return the file instead of the experiment_run in order to send it back upon request.